modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -597,6 +643,34 @@ void ONNXCompiled::setInput(int i, const cv::Mat &m) {
     preprocess(m, in_tensor_info[ort_idx], in_data[in_idx]);
 }
 
+void ONNXCompiled::setInput(ONNXCallContext &ctx,
+                            int i,
+                            int name_idx,
+                            Views& views,
+                            const cv::Rect& roi = cv::Rect()) {
+    const auto in_idx  = i;
```

Comment:
`in_idx` can be dropped I think. `i` is already a copy here, so why not use it as is

Reply:
Fixed.